NOTE: Redirects are currently not supported in Windows or MacOs.
/opt/homebrew/anaconda3/envs/uniad/lib/python3.8/site-packages/torch/distributed/launch.py:163: DeprecationWarning: The 'warn' method is deprecated, use 'warning' instead
  logger.warn(
The module torch.distributed.launch is deprecated and going to be removed in future.Migrate to torch.distributed.run
WARNING:torch.distributed.run:--use_env is deprecated and will be removed in future releases.
 Please read local_rank from `os.environ('LOCAL_RANK')` instead.
INFO:torch.distributed.launcher.api:Starting elastic_operator with launch configs:
  entrypoint       : ./tools/test.py
  min_nodes        : 1
  max_nodes        : 1
  nproc_per_node   : 8
  run_id           : none
  rdzv_backend     : static
  rdzv_endpoint    : 127.0.0.1:28596
  rdzv_configs     : {'rank': 0, 'timeout': 900}
  max_restarts     : 3
  monitor_interval : 5
  log_dir          : None
  metrics_cfg      : {}

INFO:torch.distributed.elastic.agent.server.local_elastic_agent:log directory set to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a
INFO:torch.distributed.elastic.agent.server.api:[default] starting workers for entrypoint: python
INFO:torch.distributed.elastic.agent.server.api:[default] Rendezvous'ing worker group
/opt/homebrew/anaconda3/envs/uniad/lib/python3.8/site-packages/torch/distributed/elastic/utils/store.py:52: FutureWarning: This is an experimental API and will be changed in future.
  warnings.warn(
INFO:torch.distributed.elastic.agent.server.api:[default] Rendezvous complete for workers. Result:
  restart_count=0
  master_addr=127.0.0.1
  master_port=28596
  group_rank=0
  group_world_size=1
  local_ranks=[0, 1, 2, 3, 4, 5, 6, 7]
  role_ranks=[0, 1, 2, 3, 4, 5, 6, 7]
  global_ranks=[0, 1, 2, 3, 4, 5, 6, 7]
  role_world_sizes=[8, 8, 8, 8, 8, 8, 8, 8]
  global_world_sizes=[8, 8, 8, 8, 8, 8, 8, 8]

INFO:torch.distributed.elastic.agent.server.api:[default] Starting worker group
INFO:torch.distributed.elastic.multiprocessing:Setting worker0 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_0/0/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker1 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_0/1/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker2 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_0/2/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker3 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_0/3/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker4 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_0/4/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker5 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_0/5/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker6 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_0/6/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker7 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_0/7/error.json
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
  File "./tools/test.py", line 14, in <module>
  File "./tools/test.py", line 14, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
  File "./tools/test.py", line 14, in <module>
  File "./tools/test.py", line 14, in <module>
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from mmdet3d.apis import single_gpu_test
    from mmdet3d.apis import single_gpu_test  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>

  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from mmdet3d.apis import single_gpu_test
    from mmdet3d.apis import single_gpu_test  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>

  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
        from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.apis import single_gpu_test
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
        from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,from .inference import (convert_SyncBN, inference_detector,

  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
      File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
        from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
      File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
      File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
      File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from ..structures import get_box_type    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>

  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type    
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from ..structures import get_box_type
    from .base_box3d import BaseInstance3DBoxes  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>

  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
        from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
from mmdet3d.ops.iou3d import iou3d_cuda  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>

  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
      File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from .ball_query import ball_query    
from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
        from . import ball_query_extfrom . import ball_query_ext

ImportErrorImportError: : cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)

    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
        from . import ball_query_ext    
from . import ball_query_extfrom . import ball_query_ext

ImportErrorImportError: : cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)

    ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)from . import ball_query_ext

ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: 1) local_rank: 0 (pid: 39853) of binary: /opt/homebrew/anaconda3/envs/uniad/bin/python
ERROR:torch.distributed.elastic.agent.server.local_elastic_agent:[default] Worker group failed
INFO:torch.distributed.elastic.agent.server.api:[default] Worker group FAILED. 3/3 attempts left; will restart worker group
INFO:torch.distributed.elastic.agent.server.api:[default] Stopping worker group
INFO:torch.distributed.elastic.agent.server.api:[default] Rendezvous'ing worker group
INFO:torch.distributed.elastic.agent.server.api:[default] Rendezvous complete for workers. Result:
  restart_count=1
  master_addr=127.0.0.1
  master_port=28596
  group_rank=0
  group_world_size=1
  local_ranks=[0, 1, 2, 3, 4, 5, 6, 7]
  role_ranks=[0, 1, 2, 3, 4, 5, 6, 7]
  global_ranks=[0, 1, 2, 3, 4, 5, 6, 7]
  role_world_sizes=[8, 8, 8, 8, 8, 8, 8, 8]
  global_world_sizes=[8, 8, 8, 8, 8, 8, 8, 8]

INFO:torch.distributed.elastic.agent.server.api:[default] Starting worker group
INFO:torch.distributed.elastic.multiprocessing:Setting worker0 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_1/0/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker1 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_1/1/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker2 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_1/2/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker3 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_1/3/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker4 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_1/4/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker5 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_1/5/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker6 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_1/6/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker7 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_1/7/error.json
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from mmdet3d.apis import single_gpu_test    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>

  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .ball_query import ball_query    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>

  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
        from . import ball_query_extfrom mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)

  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
Traceback (most recent call last):
    from ..structures import get_box_type  File "./tools/test.py", line 14, in <module>

  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: 1) local_rank: 0 (pid: 39932) of binary: /opt/homebrew/anaconda3/envs/uniad/bin/python
ERROR:torch.distributed.elastic.agent.server.local_elastic_agent:[default] Worker group failed
INFO:torch.distributed.elastic.agent.server.api:[default] Worker group FAILED. 2/3 attempts left; will restart worker group
INFO:torch.distributed.elastic.agent.server.api:[default] Stopping worker group
INFO:torch.distributed.elastic.agent.server.api:[default] Rendezvous'ing worker group
INFO:torch.distributed.elastic.agent.server.api:[default] Rendezvous complete for workers. Result:
  restart_count=2
  master_addr=127.0.0.1
  master_port=28596
  group_rank=0
  group_world_size=1
  local_ranks=[0, 1, 2, 3, 4, 5, 6, 7]
  role_ranks=[0, 1, 2, 3, 4, 5, 6, 7]
  global_ranks=[0, 1, 2, 3, 4, 5, 6, 7]
  role_world_sizes=[8, 8, 8, 8, 8, 8, 8, 8]
  global_world_sizes=[8, 8, 8, 8, 8, 8, 8, 8]

INFO:torch.distributed.elastic.agent.server.api:[default] Starting worker group
INFO:torch.distributed.elastic.multiprocessing:Setting worker0 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_2/0/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker1 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_2/1/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker2 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_2/2/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker3 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_2/3/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker4 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_2/4/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker5 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_2/5/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker6 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_2/6/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker7 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_2/7/error.json
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: 1) local_rank: 0 (pid: 39996) of binary: /opt/homebrew/anaconda3/envs/uniad/bin/python
ERROR:torch.distributed.elastic.agent.server.local_elastic_agent:[default] Worker group failed
INFO:torch.distributed.elastic.agent.server.api:[default] Worker group FAILED. 1/3 attempts left; will restart worker group
INFO:torch.distributed.elastic.agent.server.api:[default] Stopping worker group
INFO:torch.distributed.elastic.agent.server.api:[default] Rendezvous'ing worker group
INFO:torch.distributed.elastic.agent.server.api:[default] Rendezvous complete for workers. Result:
  restart_count=3
  master_addr=127.0.0.1
  master_port=28596
  group_rank=0
  group_world_size=1
  local_ranks=[0, 1, 2, 3, 4, 5, 6, 7]
  role_ranks=[0, 1, 2, 3, 4, 5, 6, 7]
  global_ranks=[0, 1, 2, 3, 4, 5, 6, 7]
  role_world_sizes=[8, 8, 8, 8, 8, 8, 8, 8]
  global_world_sizes=[8, 8, 8, 8, 8, 8, 8, 8]

INFO:torch.distributed.elastic.agent.server.api:[default] Starting worker group
INFO:torch.distributed.elastic.multiprocessing:Setting worker0 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_3/0/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker1 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_3/1/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker2 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_3/2/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker3 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_3/3/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker4 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_3/4/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker5 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_3/5/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker6 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_3/6/error.json
INFO:torch.distributed.elastic.multiprocessing:Setting worker7 reply file to: /var/folders/bl/c09z1sm91mxfb8z7mn7v4rhh0000gq/T/torchelastic_hx9zz20w/none_u7994h_a/attempt_3/7/error.json
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
Traceback (most recent call last):
  File "./tools/test.py", line 14, in <module>
    from mmdet3d.apis import single_gpu_test
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/__init__.py", line 2, in <module>
    from .inference import (convert_SyncBN, inference_detector,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/apis/inference.py", line 11, in <module>
    from mmdet3d.core import (Box3DMode, CameraInstance3DBoxes,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/__init__.py", line 3, in <module>
    from .bbox import *  # noqa: F401, F403
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/__init__.py", line 5, in <module>
    from .iou_calculators import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/__init__.py", line 2, in <module>
    from .iou3d_calculator import (AxisAlignedBboxOverlaps3D, BboxOverlaps3D,
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/iou_calculators/iou3d_calculator.py", line 6, in <module>
    from ..structures import get_box_type
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/__init__.py", line 2, in <module>
    from .base_box3d import BaseInstance3DBoxes
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/core/bbox/structures/base_box3d.py", line 6, in <module>
    from mmdet3d.ops.iou3d import iou3d_cuda
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/__init__.py", line 6, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py", line 1, in <module>
    from .ball_query import ball_query
  File "/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/ball_query.py", line 4, in <module>
    from . import ball_query_ext
ImportError: cannot import name 'ball_query_ext' from partially initialized module 'mmdet3d.ops.ball_query' (most likely due to a circular import) (/Users/liangming.xu/code/mmdetection3d/mmdet3d/ops/ball_query/__init__.py)
ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: 1) local_rank: 0 (pid: 40060) of binary: /opt/homebrew/anaconda3/envs/uniad/bin/python
ERROR:torch.distributed.elastic.agent.server.local_elastic_agent:[default] Worker group failed
INFO:torch.distributed.elastic.agent.server.api:Local worker group finished (FAILED). Waiting 300 seconds for other agents to finish
/opt/homebrew/anaconda3/envs/uniad/lib/python3.8/site-packages/torch/distributed/elastic/utils/store.py:70: FutureWarning: This is an experimental API and will be changed in future.
  warnings.warn(
INFO:torch.distributed.elastic.agent.server.api:Done waiting for other agents. Elapsed: 0.0017390251159667969 seconds
{"name": "torchelastic.worker.status.FAILED", "source": "WORKER", "timestamp": 0, "metadata": {"run_id": "none", "global_rank": 0, "group_rank": 0, "worker_id": "40060", "role": "default", "hostname": "MBYLJYT6J537.local", "state": "FAILED", "total_run_time": 25, "rdzv_backend": "static", "raw_error": "{\"message\": \"<NONE>\"}", "metadata": "{\"group_world_size\": 1, \"entry_point\": \"python\", \"local_rank\": [0], \"role_rank\": [0], \"role_world_size\": [8]}", "agent_restarts": 3}}
{"name": "torchelastic.worker.status.FAILED", "source": "WORKER", "timestamp": 0, "metadata": {"run_id": "none", "global_rank": 1, "group_rank": 0, "worker_id": "40061", "role": "default", "hostname": "MBYLJYT6J537.local", "state": "FAILED", "total_run_time": 25, "rdzv_backend": "static", "raw_error": "{\"message\": \"<NONE>\"}", "metadata": "{\"group_world_size\": 1, \"entry_point\": \"python\", \"local_rank\": [1], \"role_rank\": [1], \"role_world_size\": [8]}", "agent_restarts": 3}}
{"name": "torchelastic.worker.status.FAILED", "source": "WORKER", "timestamp": 0, "metadata": {"run_id": "none", "global_rank": 2, "group_rank": 0, "worker_id": "40062", "role": "default", "hostname": "MBYLJYT6J537.local", "state": "FAILED", "total_run_time": 25, "rdzv_backend": "static", "raw_error": "{\"message\": \"<NONE>\"}", "metadata": "{\"group_world_size\": 1, \"entry_point\": \"python\", \"local_rank\": [2], \"role_rank\": [2], \"role_world_size\": [8]}", "agent_restarts": 3}}
{"name": "torchelastic.worker.status.FAILED", "source": "WORKER", "timestamp": 0, "metadata": {"run_id": "none", "global_rank": 3, "group_rank": 0, "worker_id": "40063", "role": "default", "hostname": "MBYLJYT6J537.local", "state": "FAILED", "total_run_time": 25, "rdzv_backend": "static", "raw_error": "{\"message\": \"<NONE>\"}", "metadata": "{\"group_world_size\": 1, \"entry_point\": \"python\", \"local_rank\": [3], \"role_rank\": [3], \"role_world_size\": [8]}", "agent_restarts": 3}}
{"name": "torchelastic.worker.status.FAILED", "source": "WORKER", "timestamp": 0, "metadata": {"run_id": "none", "global_rank": 4, "group_rank": 0, "worker_id": "40064", "role": "default", "hostname": "MBYLJYT6J537.local", "state": "FAILED", "total_run_time": 25, "rdzv_backend": "static", "raw_error": "{\"message\": \"<NONE>\"}", "metadata": "{\"group_world_size\": 1, \"entry_point\": \"python\", \"local_rank\": [4], \"role_rank\": [4], \"role_world_size\": [8]}", "agent_restarts": 3}}
{"name": "torchelastic.worker.status.FAILED", "source": "WORKER", "timestamp": 0, "metadata": {"run_id": "none", "global_rank": 5, "group_rank": 0, "worker_id": "40065", "role": "default", "hostname": "MBYLJYT6J537.local", "state": "FAILED", "total_run_time": 25, "rdzv_backend": "static", "raw_error": "{\"message\": \"<NONE>\"}", "metadata": "{\"group_world_size\": 1, \"entry_point\": \"python\", \"local_rank\": [5], \"role_rank\": [5], \"role_world_size\": [8]}", "agent_restarts": 3}}
{"name": "torchelastic.worker.status.FAILED", "source": "WORKER", "timestamp": 0, "metadata": {"run_id": "none", "global_rank": 6, "group_rank": 0, "worker_id": "40066", "role": "default", "hostname": "MBYLJYT6J537.local", "state": "FAILED", "total_run_time": 25, "rdzv_backend": "static", "raw_error": "{\"message\": \"<NONE>\"}", "metadata": "{\"group_world_size\": 1, \"entry_point\": \"python\", \"local_rank\": [6], \"role_rank\": [6], \"role_world_size\": [8]}", "agent_restarts": 3}}
{"name": "torchelastic.worker.status.FAILED", "source": "WORKER", "timestamp": 0, "metadata": {"run_id": "none", "global_rank": 7, "group_rank": 0, "worker_id": "40067", "role": "default", "hostname": "MBYLJYT6J537.local", "state": "FAILED", "total_run_time": 25, "rdzv_backend": "static", "raw_error": "{\"message\": \"<NONE>\"}", "metadata": "{\"group_world_size\": 1, \"entry_point\": \"python\", \"local_rank\": [7], \"role_rank\": [7], \"role_world_size\": [8]}", "agent_restarts": 3}}
{"name": "torchelastic.worker.status.SUCCEEDED", "source": "AGENT", "timestamp": 0, "metadata": {"run_id": "none", "global_rank": null, "group_rank": 0, "worker_id": null, "role": "default", "hostname": "MBYLJYT6J537.local", "state": "SUCCEEDED", "total_run_time": 25, "rdzv_backend": "static", "raw_error": null, "metadata": "{\"group_world_size\": 1, \"entry_point\": \"python\"}", "agent_restarts": 3}}
/opt/homebrew/anaconda3/envs/uniad/lib/python3.8/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py:354: UserWarning: 

**********************************************************************
               CHILD PROCESS FAILED WITH NO ERROR_FILE                
**********************************************************************
CHILD PROCESS FAILED WITH NO ERROR_FILE
Child process 40060 (local_rank 0) FAILED (exitcode 1)
Error msg: Process failed with exitcode 1
Without writing an error file to <N/A>.
While this DOES NOT affect the correctness of your application,
no trace information about the error will be available for inspection.
Consider decorating your top level entrypoint function with
torch.distributed.elastic.multiprocessing.errors.record. Example:

  from torch.distributed.elastic.multiprocessing.errors import record

  @record
  def trainer_main(args):
     # do train
**********************************************************************
  warnings.warn(_no_error_file_warning_msg(rank, failure))
Traceback (most recent call last):
  File "/opt/homebrew/anaconda3/envs/uniad/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/homebrew/anaconda3/envs/uniad/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/opt/homebrew/anaconda3/envs/uniad/lib/python3.8/site-packages/torch/distributed/launch.py", line 173, in <module>
    main()
  File "/opt/homebrew/anaconda3/envs/uniad/lib/python3.8/site-packages/torch/distributed/launch.py", line 169, in main
    run(args)
  File "/opt/homebrew/anaconda3/envs/uniad/lib/python3.8/site-packages/torch/distributed/run.py", line 621, in run
    elastic_launch(
  File "/opt/homebrew/anaconda3/envs/uniad/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 116, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/opt/homebrew/anaconda3/envs/uniad/lib/python3.8/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 348, in wrapper
    return f(*args, **kwargs)
  File "/opt/homebrew/anaconda3/envs/uniad/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 245, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
***************************************
         ./tools/test.py FAILED        
=======================================
Root Cause:
[0]:
  time: 2023-06-30_00:16:16
  rank: 0 (local_rank: 0)
  exitcode: 1 (pid: 40060)
  error_file: <N/A>
  msg: "Process failed with exitcode 1"
=======================================
Other Failures:
[1]:
  time: 2023-06-30_00:16:16
  rank: 1 (local_rank: 1)
  exitcode: 1 (pid: 40061)
  error_file: <N/A>
  msg: "Process failed with exitcode 1"
[2]:
  time: 2023-06-30_00:16:16
  rank: 2 (local_rank: 2)
  exitcode: 1 (pid: 40062)
  error_file: <N/A>
  msg: "Process failed with exitcode 1"
[3]:
  time: 2023-06-30_00:16:16
  rank: 3 (local_rank: 3)
  exitcode: 1 (pid: 40063)
  error_file: <N/A>
  msg: "Process failed with exitcode 1"
[4]:
  time: 2023-06-30_00:16:16
  rank: 4 (local_rank: 4)
  exitcode: 1 (pid: 40064)
  error_file: <N/A>
  msg: "Process failed with exitcode 1"
[5]:
  time: 2023-06-30_00:16:16
  rank: 5 (local_rank: 5)
  exitcode: 1 (pid: 40065)
  error_file: <N/A>
  msg: "Process failed with exitcode 1"
[6]:
  time: 2023-06-30_00:16:16
  rank: 6 (local_rank: 6)
  exitcode: 1 (pid: 40066)
  error_file: <N/A>
  msg: "Process failed with exitcode 1"
[7]:
  time: 2023-06-30_00:16:16
  rank: 7 (local_rank: 7)
  exitcode: 1 (pid: 40067)
  error_file: <N/A>
  msg: "Process failed with exitcode 1"
***************************************

*****************************************
Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
*****************************************
